packages/runner/src/schemata/flowSettingsSchema.ts

```diff
@@ -11,19 +12,26 @@ import {
   string,
 } from "valibot";
 
-export const DEFAULT_EXCLUDE_PATTERNS = ["**/node_modules/**/*.*", "**/*.d.ts"];
```

Comment:
why we removed *d.ts from default exclude? is there use case where we want to run codemod on d.ts files?